docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -205,16 +212,42 @@ Learn more about [using Dagger with Argo Workflows](../guides/324301-argo-workfl
 </TabItem>
 </Tabs>
 
-:::tip
-Do you also want to see local runs in Dagger Cloud? Create and export a new local environment variable named `DAGGER_CLOUD_TOKEN` and set it to the value of the token obtained in [Step 1](#step-1-sign-up-for-dagger-cloud). Your local run data will now appear in Dagger Cloud.
+## Step 3: Visualize your pipeline runs with Dagger Cloud
+
+:::info
+At the end of this step, you will have data from one or more CI or local runs available for inspection and debugging in Dagger Cloud.
 :::
 
-## Step 3: Visualize a CI run with Dagger Cloud
+### Local development
+:::info Best practices for visualizing local runs in Dagger Cloud
+Dagger Cloud uses metadata from your version control system (VCS), such as git, to populate some information in the user interface. Therefore, you should run your Dagger pipelines within a VCS context, such as a local directory linked to a cloned git repository.
+:::
 
-:::info
-At the end of this step, you will have data from one or more CI runs available for inspection and analysis in Dagger Cloud.
+Before completing these steps, ensure you've [added your Dagger Cloud token as an environment variable](#local-development). 
+
+1. In your terminal, run your Dagger pipeline. If your environment variable is properly defined, you will see a Dagger Cloud URL printed in the terminal. 
+1. Click on the URL to view your pipeline run in Dagger Cloud.
+
+Here's an example running a Dagger NodeJS pipeline named index.mjs locally after committing a change with git.
+
+```shell
+  git commit -s -m "Updated quickstart link in README"
+  dagger run node index.mjs
+  ```
+
+Because we ran this in a git context, the Dagger Cloud UI shows the VCS git metadata associated with the commit. 
+
+![Local run with metadata](/img/current_docs/cloud/get-started/local-run-git-metadata.png)
+
+If you run outside of a git context, Dagger Cloud will show the same detailed steps. However, the top-level metadata will not be populated. 
+
+![Local run, no metadata](/img/current_docs/cloud/get-started/local-run-no-metadata.png)
+
+:::info Multiple Dagger Cloud organizations
```

Comment:
This is an odd place for this note. Perhaps move it to the end of the section about generating a token, or in the reference page?

Reply:
We're merging this ASAP. Let's pull any feedback into a new issue.